src/Document/TextPathElement.ts

```
@@ -0,0 +1,1369 @@
+					);
+					p1 = p2;
+				}
+
+				return len;
+
+			case 'A':
+				// Approximates by breaking curve into line segments
+				len = 0.0;
+
+				const start = points[4];
+				// 4 = theta
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)